.gitignore

```diff
@@ -8,6 +8,8 @@ metastore_db/*
 *.md5
 .Rproj.user
 .Rhistory
+py-modin
```

Comment:
`py-modin` director is expected to be location inside `modin` so this entry should not be needed if directory structure is set properly